### DISCUSSION TITLE
https://gilbertdev.net/posts/2023/02/from-disqus-to-giscus/

### DISCUSSION THREAD

**Discussion by giscus** [gilben1/gilben1.github.io#47] [Announcements]
# https://gilbertdev.net/posts/2023/02/from-disqus-to-giscus/

Commenting on changing comment backends

https://gilbertdev.net/posts/2023/02/from-disqus-to-giscus/

**Comment by gilben1** [comment]
Obligatory first comment, nice!

**Comment by arthurgousset** [comment]
Thank you for documenting your findings regarding the PaperMod theme. This [helped me](https://github.com/arthurgousset/website/pull/4) a lot!

Really appreciate it!
